recipes/baked-brie/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "baked-brie" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/UNC-CECL/brie/archive/v{{ version }}.tar.gz
+  sha256: ff48678ed9540276e216d521eeb44fd50db69fd99959c29143455cc38f34afc2
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```

I'm pretty sure this one can be noarch.

```suggestion
  number: 0
  noarch: python
```